packages/tasks/src/model-libraries.ts

```diff
@@ -358,6 +365,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"tokenizer.model"`,
 	},
+	"sae": {
+		prettyLabel: "SAE",
+		repoName: "sae",
+		repoUrl: "https://github.com/EleutherAI/sae",
+		filter: true,
+		countDownloads: `path_extension:"safetensors" OR path_extension:"bin"`,
+	},
```

Comment:
If you want, you can also provide a snippet to let the users know how to instantiate the models. The snippets are usually generated using the `modelId`, making it easy to copy-paste from the browser for end users. For instance here you could have something like:

```py
from sae import Sae

# Load specific layer
sae = Sae.load_from_hub("EleutherAI/sae-llama-3-8b-32x", layer=10)

# Load all layers
saes = Sae.load_many_from_hub("EleutherAI/sae-llama-3-8b-32x")
saes["layer_10"]
``` 

It is not necessary to explain/document everything since the SAE github repo is better suited for that but having the "Use this model" button on your repos could help getting some traction.